### ISSUE TITLE
news page for all the languages to show all the news articles

### ISSUE THREAD

**Issue by ravkiran** [hlxsites/accenture-newsroom#328]
All the sites have /news that lists all the articles along with filters

https://newsroom.accenture.sg/asia-pacific/news/
https://newsroom.accenture.pt/pt/news/
https://newsroom.accenture.de/de/news/
https://newsroom.accenture.com/news/
https://newsroom.accenture.fr/fr/news/
https://newsroom.accenture.it/it/news/
https://newsroom.accenturebr.com/br/news/
https://newsroom.accenture.es/es/news
https://newsroom.accenture.jp/jp/news
https://newsroom.accenture.co.uk/english-uk/news/

**Comment by veejayricana** [comment]
Hi @ravkiran, I think this bug https://github.com/hlxsites/accenture-newsroom/issues/334 is related to issue.

**Comment by amol-anand** [comment]
Updated all of the /news pages (all geos) manually to use the newslist block. Previewed and Published. Closing this ticket

**Comment by veejayricana** [comment]
Hi @amol-anand, I try to apply your suggestion, but the UI is different special the filter part of the page.

Here the screenshot:  
Migrate/Franklin:
![image](https://github.com/hlxsites/accenture-newsroom/assets/143703040/ca3da0b8-3272-4af8-a8b0-79c520b64b24)
Legacy:

**Comment by amol-anand** [comment]
Is there another page that has the YEAR only option or is this the only one?

**Comment by amol-anand** [comment]
Yeah but that's one page in each site. That would be inconsistent with all the other pages using the same functionality.
